src/components/Actions/Actions.js

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect } from "react";
+import { getUserDatabase } from "./../../functions/UserData";
+import { Button } from "react-bootstrap";
+import { useHistory } from "react-router";
+import Avatar1 from "./../../assets/avatars/avatar1.svg";
+import Avatar2 from "./../../assets/avatars/avatar2.svg";
+import Avatar3 from "./../../assets/avatars/avatar3.svg";
+import Avatar4 from "./../../assets/avatars/avatar4.svg";
+import WhiteCurve from "./../../assets/images/Actions/Union.svg";
+import Crown from "./../../assets/images/Actions/Crown.svg";
+import ConfettiAnimation from "./../ConfettiAnimation";
+import "./Actions.scss";
+
+const avatars = [Avatar1, Avatar2, Avatar3, Avatar4];
+
+export default function Actions() {
+  const history = useHistory();
+  const [loading, setLoading] = useState(true);
+  const [challengerInfo, setChallengerInfo] = useState(null);
+  const [ registeredVoter, setRegisteredVoter ] = useState(false);
+  const [ notifyElectionReminders, setNotifyElectionReminders ] = useState(false);
+  const [ startedChallenge, setStartedChallenge ] = useState(false);
+
+  useEffect(() => {
+    if (localStorage.getItem("challengerInfo")) {
+      setChallengerInfo(JSON.parse(localStorage.getItem("challengerInfo")));
```

Comment:
We may need to add a backup here for when there is no challenger info in the local storage (ex. user uses a different device)